src/content/docs/tutorial-optimize-telemetry/optimize-your-ingest.mdx

```diff
@@ -0,0 +1,1537 @@
+---
+title: "Optimize your data ingest"
```

Comment:
This doc has a LOT of collapsers + information. A lot of it is good, but I wonder if we should maybe unspool it further to make it more digestible?

Reply:
Putting a pin in this for future iteration discussion next week.